style.css

```diff
@@ -471,3 +472,402 @@ h2 {
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url('./Images/Icon\ -\ Mail.png');
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url("./Images/illustration-header2.png");
+    background-size: 800px 968px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headi {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-left: 235px;
+    width: 566px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: -60px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 153px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    margin-top: 150px;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 185px;
+    margin-top: 70px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(./Images/Img-Placeholder4.png);
+    background-repeat: no-repeat;
+    height: 448px;
+    width: 684px;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    justify-items: center;
+    align-items: center;
+  }
+
+  .art-container {
+    height: 435px;
+    width: 369px;
+    margin-left: 142px;
+    background-image: url("./Images/Img-Placeholder10.png");
+  }
```

Comment:
- [ ] As mentioned by the previous reviewer the "See project" button should only appear when the user hovers over the project card so please make sure to work on that. ✔
 - Normal state (without hover):
 ![image](https://user-images.githubusercontent.com/104968236/236427100-8b031d33-af67-4ace-b56b-e57205fd0ea0.png)
- When hovered:
  ![image](https://user-images.githubusercontent.com/104968236/236427204-4f42b231-643c-4724-97f9-4fefc26fafd7.png)